src/encoded/static/components/news.js

```diff
@@ -31,10 +31,12 @@ NewsPreviewItem.propTypes = {
 // Display a list of news preview items from search results.
 const NewsPreviews = (props) => {
     const { items } = props;
-    if (items && items.length) {
+    const newsItems = items && items.length ? [...items] : [];
```

Comment:
Only one little thing: this runs afoul of the [AirBnB explicit-comparison](https://github.com/airbnb/javascript#comparison--shortcuts) requirement.